README.md

```diff
@@ -1,113 +1,54 @@
-then clicking on `Edit in settings.json`. Just paste that section where appropriate.
-
-## Configuration
+![Logo](d2logo.png)
 
-The engine is configured via the `config.json` file. By default, the configuration assumes that you have installed Diablo 2 and the
-expansion via the official Blizzard Diablo 2 installers using the default file paths. If you are not on Windows, or have installed
-the game in a different location, the base path may have to be adjusted.
+[![Patreon](https://img.shields.io/badge/dynamic/json?color=%23e85b46&label=Support%20us%20on%20Patreon&query=data.attributes.patron_count&suffix=%20patrons&url=https://www.patreon.com/api/campaigns/4762180)](https://www.patreon.com/bePatron?u=37261055)
 
-## Profiling
+----
+[OpenDiablo2](https://opendiablo2.com/) is an ARPG game engine in the same vein of the 2000's games, and supports playing Diablo 2.
 
-There are many profiler options to debug performance issues. These can be enabled by suppling the following command-line option and are saved in the `pprof` directory:
+The engine is written in Go and is cross platform.
 
-`go run . --profile=cpu`
+> The project does not ship with the assets or content required to play Diablo 2.
+You must have a legally purchased copy of [Diablo 2](https://us.shop.battle.net/en-us/product/diablo-ii) and its expansion [Lord of Destruction](https://us.shop.battle.net/en-us/product/diablo-ii-lord-of-destruction) installed on your computer in order to run that game on this engine.
 
-Available profilers:\
-`cpu` `mem` `block` `goroutine` `trace` `thread` `mutex`
+If you like to contribute to OpenDiablo2, please be so kind to read our [Contribution Policy](./docs/CONTRIBUTING.md) first.
 
-You can export the profiler output with the following command:\
-`go tool pprof --pdf ./OpenDiablo2 pprof/profiler.pprof > file.pdf`
+----
 
-Ingame you can create a heap dump by pressing `~` and typing `dumpheap`. A heap.pprof is written to the `pprof` directory.
+## Documentation
 
-You may need to install [Graphviz](http://www.graphviz.org/download/) in order to convert the profiler output.
+_Stay awhile and listen_ ...
 
-## Debugging
+### ⚡ Project Info
 
-### Layouts
+* 👉 **[Current Status](./docs/status.md)** 👈 - what you should focus on
+* [Roadmap](./docs/roadmap.md) - Planning ahead
+* Design - High-level overview of the OpenDiablo2 org and its projects
+* FAQ - Common questions from new people to the project
 
-Layouts can show their boundaries and other visual debugging information when they render. Set `layoutDebug` to `true` in `d2core/d2gui/layout.go` to enable this behavior.
+### ⭐ For Users
 
-![Example layout in debug mode](https://user-images.githubusercontent.com/1004323/85792085-31816480-b733-11ea-867e-291946bfff83.png)
+* Buy the official game from Blizzard
+* Locate the MPQ files
+* Install OpenDiablo2 to your system (Linux/Windows/MacOS)
```

Comment:
let's link  `docs/install.md` here:

```md
- install [golang](https://golang.org)
- run go get github.com/OpenDiablo2/OpenDIablo2
- run go run github.com/OpenDiablo2/OpenDiablo2
- if game returns errors, check your config.json
```

Reply:
why? The end user should _not_ need to install Go, neither compile anything. When the time we should provide binaries for each $arch and $os via the GitHub releases.

Reply:
I think the plan is that a **patreon supporter** need not compile, but your average user should. DM @essial about this.

Reply:
Since I am already a Patreon supporter I can see this is indeed the case. I linked the post from the patreon page. I will also talk with essial to clarify this.